chore: add note on XRPL node spinup (#1371)
axelarnetwork/axelar-docs: src/content/docs/validator/external-chains/xrpl.mdx

curl -H 'Content-Type: application/json' -d '{"method":"server_info","params":[{}]}' http://localhost:5005
```

> ## Recovering From Crashes

Reviewer: ```suggestion
## Axelar node configuration
```

> Due to ongoing changes in the Cosmos SDK your Axelar node may run into issues when running (such as an `AppHash` error). An example of such an error may appear as follows:

Reviewer: the issue isn't ongoing changes, but due to a bug in the underlying cosmos database that causes it to fail under load such that Axelar node gets an AppHash error

```text
ERR prevote step: ProposalBlock is invalid err="wrong Block.Header.AppHash.  Expected E2B04C0FA0076BC6805192AE9659042A665B54BFB473ADE522AF582A37C847BF, got E722AD45CBD7060AC9A8A95C936113CC05BBFDC90B0091A3FBE6D0E7F7D9C93A" height=18796195 module=consensus round=0

> ERR CONSENSUS FAILURE!!! err="precommit step; +2/3 prevoted for an invalid block: wrong Block.Header.AppHash.  Expected E2B04C0FA0076BC6805192AE9659042A665B54BFB473ADE522AF582A37C847BF, got E722AD45CBD7060AC9A8A95C936113CC05BBFDC90B0091A3FBE6D0E7F7D9C93A" module=consensus stack="goroutine 136 [running]:\nruntime/debug.Stack()\n\truntime/debug/stack.go:26 +0x5e\ngithub.com/tendermint/tendermint/consensus.(*State).receiveRoutine.func2()\n\tgithub.com/tendermint/tendermint@v0.34.27/consensus/state.go:733 +0x46\npanic({0x1e76140?, 0xc12abd6260?})\n\truntime/panic.go:785 +0x132\ngithub.com/tendermint/tendermint/consensus.(*State).enterPrecommit(0xc000399508, 0x11ecea3, 0x0)\n\tgithub.com/tendermint/tendermint@v0.34.27/consensus/state.go:1421 +0x1599\ngithub.com/tendermint/tendermint/consensus.(*State).addVote(0xc000399508, 0xc02bd4fa40, {0xc00d5fe120, 0x28})\n\tgithub.com/tendermint/tendermint@v0.34.27/consensus/state.go:2153 +0x126f\ngithub.com/tendermint/tendermint/consensus.(*State).tryAddVote(0xc000399508, 0xc02bd4fa40, {0xc00d5fe120?, 0x0?})\n\tgithub.com/tendermint/tendermint@v0.34.27/consensus/state.go:1979 +0x26\ngithub.com/tendermint/tendermint/consensus.(*State).handleMsg(0xc000399508, {{0x30a1c60, 0xc1188c2998}, {0xc00d5fe120, 0x28}})\n\tgithub.com/tendermint/tendermint@v0.34.27/consensus/state.go:862 +0x15b\ngithub.com/tendermint/tendermint/consensus.(*State).receiveRoutine(0xc000399508, 0x0)\n\tgithub.com/tendermint/tendermint@v0.34.27/consensus/state.go:769 +0x3f1\ncreated by github.com/tendermint/tendermint/consensus.(*State).OnStart in goroutine 102\n\tgithub.com/tendermint/tendermint@v0.34.27/consensus/state.go:380 +0x10c\n"

Reviewer: probably don't need to show the full stack trace

Reviewer 2: just keep this part: "ERR CONSENSUS FAILURE!!! err="precommit step; +2/3 prevoted for an invalid block: wrong Block.Header.AppHash."

```

> If this error appears and your Axelar node crashes as result, then the node orchestration should perform a `rollback` on crash **before restarting the Axelar node** to auto-recover from such errors. The rollback can be performed by running:

Reviewer: ```suggestion
Your node orchestration needs to be updated to perform a `rollback` when nodes crash **before restarting the Axelar node** to auto-recover from such errors. The rollback can be performed by running:
```

> If this error appears and your Axelar node crashes as result, then the node orchestration should perform a `rollback` on crash **before restarting the Axelar node** to auto-recover from such errors. The rollback can be performed by running:

Reviewer 2: ```suggestion
Your node orchestration needs to be updated to perform a `rollback` when a node crashes. The rollback command must be executed **before restarting the Axelar node** to auto-recover from such errors. The rollback can be performed by running:
```

ampd register-chain-support amplifier xrpl
```

> ## Troubleshooting

Reviewer: This is not just trouble shooting, it must be done so should be highlighted in the steps closer to the top

> If your node runs into an issues (such as an `AppHash` error) the node orchestration should perform a `rollback` on crash **before restarting the node** to auto-recover from such errors. The rollback can be performed by running `axelard rollback`. 

Reviewer: Make it clear that it's the Axelar node that ampd points to that's being referred here, and not the XRPL node that most of this dlc focuses on

> If your node runs into an issues (such as an `AppHash` error) the node orchestration should perform a `rollback` on crash **before restarting the node** to auto-recover from such errors. The rollback can be performed by running `axelard rollback`. 

Reviewer: It's not if the node runs into an issue. It will run into an AppHash error occasionally. Provide some context on why, and that this is a temporary workaround until the underlying issue is fixed

Author: Why does this happen? I'm completely clear on this, I was reading it may be due to an issue on the cosmos sdk side, but did we want to call that out in this doc?

Reviewer 2: The issue originates from underlying Cosmo SDK. The concurrency bug happens when reading from database where the node expects to sign a specific transaction but retrieves incorrect data under heavy load. It leaves the node in a state we do not have code for handling automatically since it is unable to determine on its own what is the correct latest block

> If your node runs into an issues (such as an `AppHash` error) the node orchestration should perform a `rollback` on crash **before restarting the node** to auto-recover from such errors. The rollback can be performed by running `axelard rollback`. 

Reviewer: Show an example of the AppHash error. You can search in XRPL channel/ask protocol team
